cmd/influxd/run/server.go

```diff
@@ -383,6 +382,9 @@ func (s *Server) Close() error {
 	if s.HintedHandoff != nil {
 		s.HintedHandoff.Close()
 	}
+	if s.Monitor != nil {
```

Comment:
We seem to be moving away from putting these things under services, and then treating the like a service.  Is this a special case for some reason?